.github/workflows/test.yml

```diff
@@ -56,17 +56,16 @@ jobs:
     - name: Create SQLServer Container
       run: docker run --rm -d --name sqlserver -p 1433:1433 -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=Password12!" -e "MSSQL_COLLATION=${{ matrix.sqlserver-collation }}" -v ${GITHUB_WORKSPACE}/spec/databases/sqlserver/schema:/docker-entrypoint-initdb.d mcr.microsoft.com/mssql/server:2017-latest
 
+    - name: Create OpenSSH-Server Container
+      run: docker run --rm -d --name openssh-server -p 2222:2222 -e "DOCKER_MODS=linuxserver/mods:openssh-server-ssh-tunnel" -e "PASSWORD_ACCESS=true" -e "USER_NAME=sqlectron" -e "USER_PASSWORD=password" -e "PUBLIC_KEY_DIR=/ssh_files/pub" -v ${GITHUB_WORKSPACE}/spec/ssh_files:/ssh_files ghcr.io/linuxserver/openssh-server
+
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install -g npm
 
     - run: npm ci
-      if: matrix.node-version != '12.x'
-
-    - run: npm ci --build-from-source
-      if: matrix.node-version == '12.x'
```

Comment:
It was breaking the installation building from source. Not sure why it was required in first place, but it is working without it now.

Reply:
It was required for an older version of sqlite3 as it has a bug for certain node versions where if it failed to find a pre-compiled version, it would then kick off building from source, except that through this process, it would actually check off building from source twice simultaneously, which leads to compile time collisions and build failure. This was fixed in 5.x release, so yeah, this is not need anymore.

See: https://github.com/mapbox/node-sqlite3/issues/1329